__tests__/api/costumes.test.js

```diff
@@ -40,4 +41,28 @@ describe('GET api/costumes', () => {
         expect(response.body.costumes[1].name).toBe('buttless chaps');
         expect(response.body.costumes[2].name).toBe('bonnet');
       })
-})
\ No newline at end of file
+})
+
+
+describe('GET api/costumes/:id', () => {
+    it('returns costume by ID', async () => {
+        await createTables(pool);
+        await createCostume(pool, getBallroomGown());
+        await createCostume(pool, getButtlessChaps());
+        await createCostume(pool, getBonnet());
+        const response = await request.get('/api/costumes/2');
+        expect(response.status).toBe(200);
+        expect(response.body.costume.name).toBe('buttless chaps');
```

Comment:
Remove hard-coded strings

```suggestion
        await createCostume(pool, getBallroomGown());
        const buttlessChaps = await createCostume(pool, getButtlessChaps());
        await createCostume(pool, getBonnet());
        const response = await request.get('/api/costumes/' + buttlessChaps.id);
        expect(response.status).toBe(200);
        expect(response.body.costume.id).toBe(buttlessChaps.id);
```